### DISCUSSION TITLE
Vulnerabilities Found in Latest Keycloak Version 21.1.1

### DISCUSSION THREAD

**Discussion by pdudani09** [keycloak/keycloak#20175] [Q&A] [answered]
While running image scanning on the latest Keycloak Image using Trivy. We came across 2 vulnerabilities- a HIGH and a CRITICAL.
I have attached the report for the same. 
Can you please let us know if there is any fix that's being prepared for this ?

Regards

<img width="504" alt="Keycloak" src="https://user-images.githubusercontent.com/49002987/236392302-0b4c1e91-ba86-400d-abbc-b13d4140a3a3.PNG">

**Comment by vmuzikar** [comment]
@abstractj

**Comment by abstractj** [comment, marked as answer]
@pdudani09 Hi, CVEs fixes are planned by milestones and will be available in the upcoming releases.  For more information, please see our list of issues: https://github.com/keycloak/keycloak. If the CVE reported by the security scanner is not available on the list of our issues, it is recommended to create a new public GH issue. Dependencies with unpatched or unfixed vulnerabilities should be excluded from the report before submitting them.

Our team recently updated most of its dependency and this should be reflected by scanning the nightly images and they should be part of upcoming releases:
```
trivy image --ignore-unfixed --timeout 25m quay.io/keycloak/keycloak:nightly
```
I hope it brings some clarity.
